doc/ModulesDevelopment.md

```diff
@@ -3,42 +3,34 @@
 <!--TOC generated with https://github.com/ekalinin/github-markdown-toc-->
 <!--./gh-md-toc --insert --no-backup --hide-footer --indent 3 /path/to/README.md-->
 <!--ts-->
-* [Modules development](#modules-development)
-   * [Context](#context)
-      * [QC architecture](#qc-architecture)
-      * [DPL](#dpl)
-      * [Data Sampling](#data-sampling)
-         * [Custom Data Sampling Condition](#custom-data-sampling-condition)
-         * [Bypassing the Data Sampling](#bypassing-the-data-sampling)
-      * [Code Organization](#code-organization)
-      * [Developing with aliBuild/alienv](#developing-with-alibuildalienv)
-      * [User-defined modules](#user-defined-modules)
-      * [Repository](#repository)
-         * [Paths](#paths)
-   * [Module creation](#module-creation)
-   * [Test run](#test-run)
-      * [Saving the QC objects in a local file](#saving-the-qc-objects-in-a-local-file)
-   * [Modification of the Task](#modification-of-the-task)
-   * [Check](#check)
-      * [Configuration](#configuration)
-      * [Implementation](#implementation)
-     * [Results](#results)
-   * [Quality Aggregation](#quality-aggregation)
-      * [Quick try](#quick-try)
-      * [Configuration](#configuration-1)
-      * [Implementation](#implementation-1)
-   * [Naming convention](#naming-convention)
-   * [Committing code](#committing-code)
-   * [Data sources](#data-sources)
-      * [Readout](#readout)
-      * [DPL workflow](#dpl-workflow)
-   * [Run number and other run attributes (period, pass type, provenance)](#run-number-and-other-run-attributes-period-pass-type-provenance)
-   * [A more advanced example](#a-more-advanced-example)
-   * [Monitoring](#monitoring)
+* [Context](#context)
+   * [QC architecture](#qc-architecture)
+   * [DPL](#dpl)
+   * [Data Sampling](#data-sampling)
+   * [Code Organization](#code-organization)
+   * [Developing with aliBuild/alienv](#developing-with-alibuildalienv)
+   * [User-defined modules](#user-defined-modules)
+   * [Repository](#repository)
+* [Module creation](#module-creation)
+* [Test run](#test-run)
+* [Modification of the Task](#modification-of-the-task)
+* [Check](#check)
+   * [Configuration](#configuration)
+   * [Implementation](#implementation)
+   * [Results](#results)
+* [Quality Aggregation](#quality-aggregation)
+   * [Quick try](#quick-try)
+   * [Configuration](#configuration-1)
+   * [Implementation](#implementation-1)
+* [Naming convention](#naming-convention)
+* [Committing code](#committing-code)
+* [Data sources](#data-sources)
+   * [Readout](#readout)
```

Comment:
i guess we could remove Readout section?

Reply:
yes